packages/react-component-library/jest/setupTests.ts

```diff
@@ -1,4 +1,5 @@
 import { configure } from 'enzyme'
 import EnzymeAdapter from 'enzyme-adapter-react-16'
```

Comment:
Do we still have enzyme tests that need porting over to RTL?

Reply:
No idea, can look after the PR